torch/_subclasses/fake_tensor.py

```diff
@@ -780,7 +780,6 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         if (
             func in decomposition_table
             and torch_decomp_decompositions(func)
-            and func not in _disabled_meta_decomps
```

Comment:
@eellison check this

Reply:
@SherlockNoMad, can you find to expose the set of ops you've disabled for meta decomps to not be called ? (previously `_disabled_meta_decomps`) ?

Reply:
Here's the exhausted list. Essentially, all the decomps previous marked with "disable_meta=True" will be in this list. 

```
aten::split_with_sizes
aten::split.Tensor
aten::native_group_norm
aten::detach
aten::lift
aten::lift.out
aten::lift_fresh
aten::_adaptive_avg_pool2d
aten::_adaptive_avg_pool2d.out
aten::_reshape_alias
aten::_unsafe_view
aten::_unsafe_view.out
aten::dot
aten::dot.out
aten::expand
aten::permute
aten::index_select
aten::index_select.out
aten::index_select.dimname
aten::index_select.dimname_out
aten::squeeze
aten::squeeze.dim
aten::squeeze.dimname
aten::diagonal
aten::diagonal.Dimname
aten::t
aten::transpose.int
aten::transpose.Dimname
aten::unsqueeze
aten::view
aten::view.dtype
```

Reply:
Yea, we can we avoid calling those decopmositions here in fake tensor, as we were before ?